python/vyos/frrender.py

```diff
@@ -234,13 +234,15 @@ def dict_helper_nhrp_defaults(nhrp):
             dict.update({ip_version : ip_dict})
 
     # Get FRR profile
-    frr_defaults_cli_path = ['system', 'frr', 'profile']
-    if conf.exists(frr_defaults_cli_path):
-        dict['frr'] = {
-            'defaults': conf.get_config_dict(
-                frr_defaults_cli_path, key_mangling=('-', '_')
-            )['profile']
-        }
+    frr_system_cli_path = ['system', 'frr']
```

Comment:
Can you please follow the existing code style:

<img width="1057" height="133" alt="image" src="https://github.com/user-attachments/assets/8852f712-3f2b-4d53-91e2-fe82cd38cc94" />